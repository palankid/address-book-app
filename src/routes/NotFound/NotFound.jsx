import React from 'react';

const NotFound = () => {
  return (
    <div className="not-found">
      <span className="not-found__content">
        Error 404: Page not found
      </span>
    </div>
  );
};

export default NotFound;
