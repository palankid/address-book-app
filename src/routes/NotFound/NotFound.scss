#app {
  height: 100%;
}

.not-found {
  height: 100%;
  display: flex;
  align-items: center;
  justify-content: center;

  &__content {
    font-size: 2rem;
  }
}
