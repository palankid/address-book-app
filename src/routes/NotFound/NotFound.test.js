import React from 'react';
import { shallow } from 'enzyme';

import NotFound from './NotFound';

describe('NotFound', () => {
  it('should test that component is not changed unintentionally', () => {
    const wrapper = shallow(<NotFound />);

    expect(wrapper).toMatchSnapshot();
  });
});
