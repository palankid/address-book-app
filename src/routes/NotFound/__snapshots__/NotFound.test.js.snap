// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LoadingModule should test that component is not changed unintentionally 1`] = `
<div
  className="not-found"
>
  <span
    className="not-found__content"
  >
    Error 404: Page not found
  </span>
</div>
`;
